EIPS/eip-7528.md

```diff
@@ -0,0 +1,157 @@
+---
+eip: 7528
```

Comment:
```suggestion
eip: 7529
```
Assigning next sequential EIP/ERC number.  7528 already assigned.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
thank you, on it